content/cloud-docs/workspaces/creating.mdx

```diff
@@ -34,48 +31,49 @@ New workspaces can be created by teams with permission to manage workspaces. ([M
+1. (Version control only) Choose a repository from the filterable list. The **Configure settings** page appears.
 
-1. [Navigate to the workspace list](/cloud-docs/workspaces/#listing-and-filtering-workspaces) and click the "+ New Workspace" button (near the top of the page).
+  -> **Note:** Depending on your provider, you may need to change which account's repositories are shown. If the repository is not listed, scroll to the bottom of the list and enter its ID in the text field.
 
-1. On the first screen, choose your VCS provider (or choose "No VCS connection").
+1. Enter a **Workspace Name**. This defaults to the repository name, if applicable. The name must be unique within the organization and can include letters, numbers, dashes (`-`), and underscores (`_`). Refer to our [workspace naming recommendations](/cloud-docs/workspaces/naming).
+1. Add an optional **Description** that will appear at the top of the workspace in the Terraform Cloud UI.
 
-   -> **Note:** If you haven't added a VCS provider for your organization yet, choosing one here will prompt you to configure it. See [Connecting VCS Providers](/cloud-docs/vcs) for more information.
+1. (Version control only) Open **Advanced options** to optionally configure the following additional settings:
+   - [Terraform Working Directory][workdir]
+   - [Automatic Run Triggering][trigger]
+   - [VCS branch][branch]
+   - [Include submodules on clone][submodules]
 
-1. On the second screen, choose a repository from the filterable list. This screen is skipped if you chose "No VCS connection".
+1. Click **Create workspace**.
 
-   Some VCS providers limit the list's size. If a repository isn't listed, you can still choose it by name; scroll to the bottom of the list and enter its ID in the text field.
+For CLI and API-driven workspaces, the workspace overview page appears. For version control workspaces, the **Configure Terraform variables** page appears.
 
-   -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
+### Configure Terraform Variables (Version Control Only)
 
-1. On the third screen, enter a name for the workspace. This defaults to the repository name, if applicable. The name must be unique within the organization, and can include letters, numbers, dashes (`-`), and underscores (`_`). See also our [advice for useful workspace names](/cloud-docs/workspaces/naming).
+After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set values for these undefined variables.
```

Comment:
There is also a nuance with working directories at the moment. Maybe at the end something like:

```suggestion
After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set values for these undefined variables.

This new variable detection feature currently only works if the terraform config is in the root (default) working directory. If you are using a different working directory, you can enter and configure the variables manually from the workspace overview.
```

Reply:
To include what Laura said above


```suggestion
After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](https://github.com/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set values for these variables."

This new variable detection feature currently only works if the terraform config is in the root (default) working directory. If you are using a different working directory, you can enter and configure the variables from the workspace overview.
```

Reply:
This is really great to know! I wonder if we could add this as a note - as we do later on in the docs. Let me take a look at this 😄

Reply:
ooh -- if there's a notes section that sounds good!